test/unit/Services/DepositTransactionService.spec.ts

```diff
@@ -0,0 +1,10 @@
+import {generateDeposit} from "../../../src/renderer/services/DepositTransactionService";
+import {Keypair as KeyPair} from "@chainsafe/bls/lib/keypair";
+
+describe("Deposit transaction service unit tests", () => {
+
+    it("Generate deposit params successfully", () => {
+        const res = generateDeposit(KeyPair.generate(), Buffer.alloc(48, 1,"hex"));
```

Comment:
test will be faster if you use `new Keypair(PrivateKey.fromBytes(new BN(123).toArrayLike(Buffer, 'le', 32))` as it will not generate entropy and stuff. You should extract this as util method